### PR TITLE
fix: Author Rendering at the Blog page 

### LINKED ISSUE #1065: 🐛 Bug: Blog Page Author rendering problem
### Describe the bug

The Authors on the Blog page are not rendering as expected.

### Steps To Reproduce

1. Changing the Styles on the blogs page.
2. Making some changes in tailwind.config file.blog

### Expected Behavior

Correct rendering of all authors for their respective blogs.

### Screenshots

Screenshot 2024-10-26 at 2 43 31 PM

### Device Information [optional]

```markdown
- OS: MacOS
- Browser: Chrome
```

### Are you working on this issue?

Yes

### Do you think this work might require an [Architectural Decision Record (ADR)]? (significant or noteworthy)

Yes

### PR DESCRIPTION
**Pull request by Ritesh2351235**
**What kind of change does this PR introduce?**
Changing the styling in the Blog page to avoid conflicts with the PurgeCSS.

**Issue Number:**
-  Closes #1065

### CODE REVIEW

File: _includes/community
Comment (DhairyaMajmudar) [inline _includes/community:1]: Pls. remove changes from here

**Review by DhairyaMajmudar — CHANGES_REQUESTED** [review]
@Ritesh2351235 pls. remove the changes associated with /community folder. Here its used as git submodule thus if you wanna perform any change in it you have to open a PR in https://github.com/json-schema-org/community/  repo

**Comment by Ritesh2351235** [comment]
@DhairyaMajmudar @benjagm 
Hey Benja, I'm actually stuck on this problem. I tried avoiding string concatenation and adding a safelist to the Tailwind config files. Are there any other ways I could solve this problem?

**Comment by Ritesh2351235** [comment]
@DhairyaMajmudar I think the changes are working fine now.

**Comment by DhairyaMajmudar** [comment]
@Ritesh2351235 is earlier commit the changes were related to some other lines of code. Can you pls. check that.

**Comment by Ritesh2351235** [comment]
@DhairyaMajmudar I have removed that changes there was a problem while rebasing the repo 
Now I have only updated the blog page

**Comment by Ritesh2351235** [comment]
@DhairyaMajmudar Can you just check with the production code theater the authors are rendering correctly in the blog page?
Let me know if any changes are required.

**Review by DhairyaMajmudar — COMMENTED** [review]
```tsx
className={`
                                      bg-slate-50
                                      rounded-full
                                      -ml-3
                                      bg-cover
                                      bg-center
                                      border-2
                                      border-white 
                                      ${sizeClass}`}
```

**Comment by DhairyaMajmudar** [comment]
> @DhairyaMajmudar Can you just check with the production code theater the authors are rendering correctly in the blog page? Let me know if any changes are required.

True @Ritesh2351235 on production website the bug is now not showing but its seems like the bug mentioned arises in some special cases. The styles affecting the authors' images are joined using array methods which is quite strange. 
Let's replace them with the string literals as I've shown in this comment https://github.com/json-schema-org/website/pull/1066#pullrequestreview-2398167388

**Comment by benjagm** [comment]
I think the styles are not working because of the Tailwind styles purge process. Specifically, we should avoid using string concatenation to create class names:

https://v2.tailwindcss.com/docs/optimizing-for-production#writing-purgeable-html


See https://github.com/json-schema-org/website/pull/1029#pullrequestreview-2379320382

**Comment by Ritesh2351235** [comment]
@DhairyaMajmudar Went through all the Documentation and this should solve the problem now !!

**Review by DhairyaMajmudar — APPROVED** [review]
LGTM, thank you for the PR @Ritesh2351235 🚀

**Review by benjagm — APPROVED** [review]
LGTM

**Pull request merged**